process/mock/scQueryMock.go

```diff
@@ -18,6 +22,11 @@ func (s *ScQueryMock) ExecuteQuery(query *process.SCQuery) (*vmcommon.VMOutput,
 	return &vmcommon.VMOutput{}, nil
 }
 
+// ComputeTransactionCost --
+func (s *ScQueryMock) ComputeTransactionCost(tx *transaction.Transaction) (*big.Int, error) {
+	return s.ComputeTransactionCostHandler(tx)
```

Comment:
Refactor to:
```GOLANG
if s.ComputeTransactionCostHandler != nil {
  return s.ComputeTransactionCostHandler(tx)
}
return big.NewInt(X), nil
```

Also in the other similar stubs you edited in this PR

Reply:
done.